.github/workflows/python-package-conda.yml

```
@@ -0,0 +1,34 @@
+name:simran Python Package using Conda
+
+on: [push]
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 3: mapping values are not allowed in this context

(syntax-check)